puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -23,6 +23,7 @@
   }
 
   cron { "jenkins-jobs-update-${config_name}-delete-old":
```

Comment:
I won't miss this daily email from cron, but what is now responsible for deleting old (unmanaged) jobs?

Reply:
Is an unmanaged job a job created out of band of JJB?

Reply:
we call `update --delete-old` which removes all jobs that are not currently defined in jjb

Reply:
Can be. It also deleted jobs that were removed from our JJB config.